eidas-saml-parser/src/test/java/uk/gov/ida/notification/eidassaml/apprule/EspAuthnRequestAppRuleTest.java

```diff
@@ -1,30 +1,40 @@
 package uk.gov.ida.notification.eidassaml.apprule;
 
-import org.opensaml.core.xml.io.MarshallingException;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
 import org.opensaml.saml.saml2.core.AuthnRequest;
 import uk.gov.ida.notification.contracts.EidasSamlParserResponse;
 import uk.gov.ida.notification.eidassaml.apprule.base.EidasSamlParserAppRuleTestBase;
 import uk.gov.ida.notification.helpers.EidasAuthnRequestBuilder;
+import uk.gov.ida.notification.pki.X509CredentialFactory;
+import uk.gov.ida.notification.saml.SamlObjectSigner;
 
 import javax.ws.rs.core.Response;
-import javax.xml.transform.TransformerException;
-import javax.xml.xpath.XPathExpressionException;
 
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.fail;
+import static uk.gov.ida.saml.core.test.TestCertificateStrings.TEST_RP_PRIVATE_SIGNING_KEY;
+import static uk.gov.ida.saml.core.test.TestCertificateStrings.TEST_RP_PUBLIC_SIGNING_CERT;
 
 public class EspAuthnRequestAppRuleTest extends EidasSamlParserAppRuleTestBase {
 
     private EidasAuthnRequestBuilder request;
 
+    @Before
     public void setup() throws Throwable {
         request = new EidasAuthnRequestBuilder().withIssuer(CONNECTOR_NODE_ENTITY_ID);
     }
 
-    public void shouldReturnRequestIdAndIssuer() throws XPathExpressionException, MarshallingException, TransformerException{
+    @Test
+    @Ignore
+    public void shouldReturnRequestIdAndIssuer() throws Exception {
```

Comment:
There's a ticket to look into the ESP integration tests which I think Dan may have started so should be resolved soon.

Reply:
:+1: